test/shared/createElement.test.js

```diff
@@ -44,8 +44,8 @@ describe('createElement(jsx)', () => {
 	});
 
 	it('should set VNode#key property', () => {
-		expect(<div />).to.have.property('key').that.is.undefined;
-		expect(<div a="a" />).to.have.property('key').that.is.undefined;
+		expect(<div />).to.have.property('key').that.is.empty;
+		expect(<div a="a" />).to.have.property('key').that.is.empty;
```

Comment:
I think these assertions want to be `.to.not.exist`  which would pass for `null` or `undefined`

Reply:
whoops, I literally read that in the docs and then wrote `.empty`. Tired me is tired!